controller/internal/enforcer/nfqdatapath/nfq_linux.go

```diff
@@ -103,14 +107,14 @@ func (d *Datapath) processNetworkPacketsFromNFQ(p *nfqueue.NFPacket) {
 		)
 		length := uint32(len(p.Buffer))
 		buffer := p.Buffer
-		p.QueueHandle.SetVerdict2(uint32(p.QueueHandle.QueueNum), 0, uint32(p.Mark), length, uint32(p.ID), buffer)
+		p.QueueHandle.SetVerdict2(0, uint32(p.Mark), length, uint32(p.ID), buffer)
 		if netPacket.IPProto() == packet.IPProtocolTCP {
```

Comment:
why 0 ? doesn't look right

Reply:
I removed the first argument. QueueNum is not required in SetVerdict function.